# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../agg/no-examples.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- agg-2.5/Makefile.am.vanilla	2018-11-18 18:06:17.572937986 +0000
+++ agg-2.5/Makefile.am	2018-11-18 18:06:21.696937986 +0000
@@ -1,4 +1,4 @@
-SUBDIRS = src font_freetype font_win32_tt include examples
+SUBDIRS = src font_freetype font_win32_tt include
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libagg.pc
